designsafe/apps/api/licenses/views.py

```diff
@@ -0,0 +1,48 @@
+from designsafe.apps.api.views import BaseApiView
+from designsafe.apps.api.mixins import SecureMixin
+from designsafe.libs.common.decorators import profile as profile_fn
+from django.contrib.auth import get_user_model
+from django.http.response import HttpResponseForbidden
+from django.http import JsonResponse
+from django.apps import apps
+from designsafe.apps.licenses.models import MATLABLicense, LSDYNALicense
+from designsafe.apps.data.models.agave.util import AgaveJSONEncoder
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+
+class MATLABLicenseView(SecureMixin, BaseApiView):
+    @profile_fn
+    def get(self, request, username):
+        if not request.user.is_superuser:
+            return HttpResponseForbidden()
```

Comment:
What's the use case for this again? Cause this would prevent anyone but superusers (i.e. our development team) from using this endpoint

Reply:
I was thinking this would be similar to the projects endpoint we have in DS as well where superusers can imitate any user and grab the desired info (projects, licenses) for use in Jupyter, so we can run apps which require licenses through Jupyter.